csp/gitprojectsettings.csp

```diff
@@ -134,7 +155,13 @@ body {
     do settings.%Save()
 }
 </server>
-<div class = 'container'>    
+<div class = 'container'>
+    <csp:if condition='$D(%request.Data("gitsettings",1)) && (##class(SourceControl.Git.Utils).NeedSettings() = 0)'>
+        <div class = "alert" style="display: 'none'">
```

Comment:
Why does this have display: 'none'? Note, I don't think this is valid anyway, should be display: none (but I think you really just don't want it to be there at all)